### ISSUE TITLE
loot white screen

### ISSUE THREAD

**Issue by Sylfelin** [loot/loot#1404]
Hi,
in 0.16.0 is ok
since version loot_0.16.0-52-gbd3112a  I have a white screen. I can't test with a more recent version.

I try all sollution find in forum
--game, %LOCALAPPDATA%\LOOT, cleaning driver card, etc... (read all thread)

I have attached the file LOOTDebugLog.txt
there seems to be a problem on the way

[LOOTDebugLog.txt](https://github.com/loot/loot/files/5786383/LOOTDebugLog.txt)

thank's

**Comment by Ortham** [comment]
```
[10:37:15.220042] [info]: Handling request to URL: http://loot/ui/auto/app.bundle.js
[10:37:15.220127] [trace]: File D:\_PL_GAME\Tools\LOOT\resources\ui/auto/app.bundle.js not found, sending 404.
```

That looks odd, I didn't expect the URL/path to contain `/auto`. Can you share the debug log that 0.16.0 creates for comparison?

**Comment by Sylfelin** [comment]
my  debug log for 0.16.0

[LOOTDebugLog.txt](https://github.com/loot/loot/files/5811397/LOOTDebugLog.txt)

**Comment by Ortham** [comment]
Fixed by da878f485e68afdb43f3af9041f8296e5c7e77d6.
